prov/efa/src/efa_fabric.c

```diff
@@ -496,7 +523,7 @@ static int efa_get_device_attrs(struct efa_context *ctx, struct fi_info *info)
 	struct efa_device_attr device_attr;
 	struct ibv_device_attr *base_attr;
 	struct ibv_port_attr port_attr;
-	int ret;
+	int gdr_support, ret;
```

Comment:
needs to be wrapped in `#if HAVE_LIBCUDA`

Reply:
Thank you fixed in new revision.